internal/sql/info/info.go

```diff
@@ -0,0 +1,59 @@
+package info
+
+import (
+	"github.com/kyleconroy/sqlc/internal/sql/catalog"
+)
+
+// Provide a read-only view into the catalog
+func Newo(c *catalog.Catalog) InformationSchema {
```

Comment:
Newo?

Reply:
Oops, as I said, that package is experimental and not used.